CDN/upgrade.sh

```diff
@@ -268,10 +268,12 @@ then
 	#run_cmd cp -r $MOUNT_PATH$TARGET/$CDNROOT-$NVER $TARGET
 	run_cmd  mv $TARGET/$CDNROOT $TARGET/$CDNROOT.backup
 	run_cmd ln -s $MOUNT_PATH$TARGET/$CDNROOT-$NVER $TARGET/$CDNROOT
-	run_cmd "cp -frs $TARGET/$ROOTFS_OVERLAY/* /$MOUNT_PATH/"
+	#run_cmd "cp -frs $TARGET/$ROOTFS_OVERLAY/* /$MOUNT_PATH/"
+    run_cmd "find $TARGET/$ROOTFS_OVERLAY/ -mindepth 1 -maxdepth 1 -name var -prune -o -exec cp -frst /$MOUNT_PATH/ {} \;"
```

Comment:
Why is this change ? Need a reason..

Reply:
The requirement was to [keep the captive portal intact on an upgrade](https://github.com/projectOpenRAP/OpenRAP/issues/250), but I realized that this would also prevent any additional plugin UI from being setup, for e.g. `var/www/gok` will also be skipped over. So this will have to be changed.